cypress/e2e/tests/migration/applicationinventory/assessment/create.test.ts

```diff
@@ -45,24 +44,26 @@ import {
     applicationDescriptionInput,
     applicationNameInput,
     applicationBusinessServiceSelect,
-    actionButton,
+    applicationContributorsInput,
 } from "../../../../views/applicationinventory.view";
 
 import * as commonView from "../../../../views/common.view";
 import * as data from "../../../../../utils/data_utils";
 import { BusinessServices } from "../../../../models/migration/controls/businessservices";
 import { Assessment } from "../../../../models/migration/applicationinventory/assessment";
+import { Stakeholders } from "../../../../models/migration/controls/stakeholders";
 
-var businessservicesList: Array<BusinessServices> = [];
-var applicationList: Array<Assessment> = [];
+let businessservicesList: Array<BusinessServices> = [];
+let applicationList: Array<Assessment> = [];
+let stakeHoldersList: Stakeholders[];
 
 describe(["@tier2"], "Application validations", () => {
     before("Login", function () {
         // Perform login
         login();
         deleteApplicationTableRows();
-        applicationList = createMultipleApplications(11);
         businessservicesList = createMultipleBusinessServices(1);
+        stakeHoldersList = createMultipleStakeholders(2);
```

Comment:
LGTM . 

Let's clean up stakeholders in after() method .

Reply:
Done